src/main/haskell/language-kore/test/parser/Data/Kore/Parser/ParserTest.hs

```diff
@@ -788,18 +788,19 @@ variablePatternParserTests =
                 { variableName = testId "v" :: Id Object
                 , variableSort =
                     SortActualSort SortActual
-                        { sortActualName=testId "s1"
-                        , sortActualSorts = [ sortVariableSort "s2" ]
-                        }
+                    { sortActualName=testId "s1"
```

Comment:
either indent this as it was, or move the SortActualSort SortActual on the previous line